test/models/ocelot/LabelSpec.scala

```diff
@@ -18,31 +18,45 @@ package models.ocelot
 
 import base.BaseSpec
 import play.api.libs.json._
-import models.ocelot.stanzas._
 
 class LabelSpec extends BaseSpec with ProcessJson {
 
-
   val label = """{"name":"BLAH"}"""
-  val labelWithValue = """{"name":"BLAH","value":"39.99"}"""
-  val labelWithType = """{"name":"BLAH","valueType":"Currency"}"""
-  val labelWithValueAndType = """{"name":"BLAH","value":"32.99","valueType":"Currency"}"""
+  val labelWithSingleValue = """{"name":"BLAH","english":"39.99"}"""
+  val labelWithValues = """{"name":"BLAH","english":"Hello","welsh":"Welsh, Hello"}"""
 
   "Label" must {
     "deserialise " in {
       Json.parse(label).as[Label] shouldBe Label("BLAH")
-      Json.parse(labelWithValue).as[Label] shouldBe Label("BLAH", Some("39.99"))
-      Json.parse(labelWithType).as[Label] shouldBe Label("BLAH", None, Some(Currency))
-      Json.parse(labelWithValueAndType).as[Label] shouldBe Label("BLAH", Some("32.99"), Some(Currency))
+      Json.parse(labelWithSingleValue).as[Label] shouldBe Label("BLAH", Some("39.99"))
     }
 
     "serialise from Label to json" in {
-      Json.toJson(Label("BLAH")).toString shouldBe label
-      Json.toJson(Label("BLAH", Some("39.99"))).toString shouldBe labelWithValue
-      Json.toJson(Label("BLAH", None, Some(Currency))).toString shouldBe labelWithType
-      Json.toJson(Label("BLAH", Some("32.99"), Some(Currency))).toString shouldBe labelWithValueAndType
+      val valueLabel: Label = Label("BLAH")
+      Json.toJson(valueLabel).toString shouldBe label
+      val valueLabelWithValue: Label = Label("BLAH", Some("39.99"))
+      Json.toJson(valueLabelWithValue).toString shouldBe labelWithSingleValue
+    }
+
+    "deserialise label with both english and welsh values" in {
+      Json.parse(labelWithValues).as[Label] shouldBe Label("BLAH", Some("Hello"), Some("Welsh, Hello"))
     }
 
+    "serialise from Label with both english and welsh values to json" in {
+      val dLabelWithValues: Label = Label("BLAH", Some("Hello"), Some("Welsh, Hello"))
+      Json.toJson(dLabelWithValues).toString shouldBe labelWithValues
+    }
+
+  }
+
+  "Label" must {
```

Comment:
You have repeated the test name. I would combine the tests or create a new name if that is more appropriate.

Reply:
will fix